pkg/reftracker/ref_key.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2021 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package reftracker
+
+type RefKey struct {
+	kind, refName, namespace string
+}
+
+func NewRefKey(kind, refName, namespace string) RefKey {
```

Comment:
might be convenient to add NewConfigMapRefKey and NewSecretRefKey constructors

Reply:
Add for those and apps.